gnosis/gnosis-spec.ini

```diff
@@ -277,7 +277,10 @@ proxy_storage: _
     andBool G >=Int 10000 //
 
 [encodeTransactionData]
-output: _ => _
+output: _ => #encodeArgs( #bytes(
+                #parseHexWord("0x19") : #parseHexWord("0x1")
+                : #encodeArgs(#bytes32(DOMAIN_SEPARATOR), #bytes32(SAFE_TX_HASH))
+             ))
 statusCode: _ => EVMC_SUCCESS
```

Comment:
Does `encodeTransactionData` always succeed?

Reply:
The specification passes. So yes.